internal/server/grpc/grpc.go

```diff
@@ -101,3 +150,14 @@ func (s *Server) Stop() error {
 
 	return nil
 }
+
+func loadCertFile(certFile string) ([]byte, error) {
```

Comment:
Suggesting

```go
func loadCertFile(certFile string) ([]byte, error) {
	if certFile == "" {
		return nil, fmt.Errorf("cert file is empty")
	}

	return os.ReadFile(certFile)
}

```

Reply:
@rodneyosodo , 
In server, if the CertFile is not provided, then it should automatically switch to non-tls http
So if there is no file it should ignore and run http server 

I have modified like as given below
```go
func loadCertFile(certFile string) ([]byte, error) {
	var dd []byte
	if certFile != "" {
		var err error
		dd, err = ioutil.ReadFile(certFile)
		if err != nil {
			return nil, err
		}
	}
	return dd, nil
}
```

Reply:
What about something like this
```go
func loadCertFile(certFile string) ([]byte, error) {
	var dd []byte
	if certFile != "" {
		return os.ReadFile(certFile)
	}
	return dd, nil
}
```